pkg/k8s/apis/cilium.io/const.go

```diff
@@ -21,9 +21,12 @@ import (
 const (
 	// PolicyLabelName is the name of the policy label which refers to the
 	// k8s policy name.
-	PolicyLabelName = "io.cilium.k8s-policy-name"
+	PolicyLabelName = "io.cilium.k8s.policy.name"
```

Comment:
I've thought about it, I don't think so because this is used on each local cilium policy repository. If this value is the same inside each cilium agent then it's fine.